blogs/2022-06-28-tips-kafka-streams-distributed.md

```diff
@@ -134,29 +134,29 @@ builder.addGlobalStore(
     )
 );
 ```
-with [GlobalInMemoryStateProcessor](https://github.com/kestra-io/kestra/blob/master/runner-kafka/src/main/java/io/kestra/runner/kafka/streams/GlobalInMemoryStateProcessor.java) a simple wrapper that saves to state store and sends a full list on every change (not so frequent) and decided to gather all flow in memory. This works well on our use cases because we know that an instance of Kestra will not have millions of flows.
+We accomplish this with [GlobalInMemoryStateProcessor](https://github.com/kestra-io/kestra/blob/master/runner-kafka/src/main/java/io/kestra/runner/kafka/streams/GlobalInMemoryStateProcessor.java), a simple wrapper that saves to state store and sends a complete list on every change (even though this doesnt happen frequently) and gathers all flow in memory. This works well in our use case because we know that an instance of Kestra will not have millions of flows.
 
-But keep in mind, all store operations (also get) will lead to deserialization that costs you some CPU.
+But, it's important to remember that all store operations will lead to deserialization that costs you some CPU.
 
 
 ## Many source topics within a stream
```

Comment:
You could also add a sentence or tow here introducing the sub-header and putting it in the context of the entire article.